### DISCUSSION TITLE
Pesan Tidak Terkirim Meski Statusnya Success

### DISCUSSION THREAD

**Discussion by melodiazz** [kirimwa-id/forum#123] [Tanya Jawab]
Hi Pak Rio,

Mau informasikan bahwa 2 hari terakhir cukup banyak pesan yang tidak terkirim meski setelah saya cek status messagenya success (bukan fail / pending).
Saya kroscek juga status devicenya masih connected baik pada Status Kirimwa.id maupung status linked devices pada android.

**Comment by shahib-mirbath** [comment]
Saya juga. Mengalami itu saat kirim batch file pdf atau notif. Di web wa ada historinya tp di wa device tidak ada.

**Reply by rioastamal** [reply]
@melodiazz @shahib-mirbath bisa kirimkan sample message_id dari pesan yang tidak sampai?

**Reply by melodiazz** [reply]
berikut pak Rio kwid-9884bffba49a428187146774198

**Comment by valegilbert** [comment]
Kalau kasus saya, pesan di wa web statusnya terkirim dan diterima oleh whatsapp penerima, tapi di whatsapp pengirim, pada device android, pesan tersebut tdk muncul...

**Reply by shahib-mirbath** [reply]
Sama. Saya juga dapat kasus seperti ini

**Reply by rioastamal** [reply]
Kalau ini sepertinya masalah sinkronisasi state yang bermasalah atau tertunda.

**Comment by melodiazz** [comment]
Pagi ini setelah sempat dapat status 503 beberapa menit, setelah itu seluruh pesan keluar (yang incoming message saya kurang tau karna belum pernah pakai webhook) lancar jaya.. thanks pak Rio
